### PR TITLE
Added xkcd-notifier script

### PR DESCRIPTION
**Pull request by kippenjongen**
Adds a script that displays the current xkcd comic number and whether there are any new ones.
(Sorry for the previous pull request, I realized it didn't quite work as it should yet)

### CODE REVIEW

File: polybar-scripts/xkcd-notifier/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Script: xkcd-notifier
```
Comment (x70b1) [inline polybar-scripts/xkcd-notifier/README.md:1]: Lets rename the script, directory and other descriptions to `notification-xkcd`.

File: polybar-scripts/xkcd-notifier/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Displays the current xkcd comic and whether there are any new ones in polybar, written in python.
+
+![Unread](./screenshots/unread.png) ![Read](./screenshots/read.png)
```
Comment (x70b1) [inline polybar-scripts/xkcd-notifier/README.md:5]: Rename the screenshots to `1.png` and `2.png`,

File: polybar-scripts/xkcd-notifier/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Dependencies
+
+- Python 3
```
Comment (x70b1) [inline polybar-scripts/xkcd-notifier/README.md:9]: This is obvious, we can remove this part.

File: polybar-scripts/xkcd-notifier/notifier.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+from sys import argv
+
+latest_file = "latest"
+prefix = "#"
+read = False
+
+help = """xkcd-notifier: Display the current XKCD comic number, and show if there are any new ones.
+
+Usage:
+	-h, --help	Display this message
```
Comment (x70b1) [inline polybar-scripts/xkcd-notifier/notifier.py:13]: Actually I would not use arguments. I would use directly variables, which can be modified by everyone. 
This makes the script shorter. But that is your decision here.

**Review by x70b1 — CHANGES_REQUESTED** [review]

File: polybar-scripts/xkcd-notifier/notifier.py (hunk 4 above)
Comment (kippenjongen) [reply polybar-scripts/xkcd-notifier/notifier.py]: That sounds like a good idea, I've done that

**Comment by kippenjongen** [comment]
I've added the changes requested and requested a new review, is that good? (This is my first pull request, I'm not quite sure how it works yet, sorry)

**Comment by x70b1** [comment]
@kippenjongen Very nice! :sunglasses:

**Review by x70b1 — APPROVED** [review]

**Pull request merged**